cmake/LLVMIRUtil.cmake

```diff
@@ -166,25 +166,27 @@ function(llvmir_attach_bc_target)
 
   # main operations
   foreach(IN_FILE ${IN_FILES})
+    get_target_property(SOURCE_DIR ${DEPENDS_TRGT} SOURCE_DIR)
+    get_filename_component(ABS_IN_FILE ${IN_FILE} ABSOLUTE BASE_DIR ${SOURCE_DIR})
```

Comment:
Get the absolute path at the start of each loop, and replace all usage of `IN_FILE` and `INFILE` with `ABS_IN_FILE`.

According to the document and some simple tests, the results are the same when using `IN_FILE` and `ABS_IN_FILE`, either for getting ext `get_filename_component(FILE_EXT ${ABS_IN_FILE} LAST_EXT)` or for getting file name `get_filename_component(OUTFILE ${ABS_IN_FILE} NAME)`.

Reply:
LGTM, please combine the commits and I'll merge it in.